tests/h/views/admin_users_test.py

```diff
@@ -342,3 +345,26 @@ def annotation_stats_service(pyramid_config, db_session):
 def user_created_no_groups(models):
     # By default, pretend that all users are the creators of 0 groups.
     models.Group.created_by.return_value.count.return_value = 0
+
+
+@pytest.fixture
+def group_with_two_users(db_session, factories):
+    """
+    Create a group with two members and an annotation created by each.
+    """
+    user = factories.User()
+    other_user = factories.User()
+
+    group = Group(authority=user.authority, creator=user, members=[user, other_user],
+                  name='test', pubid='group_with_two_users')
+    db_session.add(group)
+
+    doc = Document(web_uri='https://example.org')
+    user_ann = Annotation(userid=user.userid, groupid=group.pubid, document=doc)
+    other_user_ann = Annotation(userid=other_user.userid, groupid=group.pubid,
+                                document=doc)
+    db_session.add(user_ann)
+    db_session.add(other_user_ann)
+    db_session.flush()
```

Comment:
It's not a problem that was introduced by this pull request, but I think code and tests that access the DB should be in the groups model and service, not in views. I guess this is a refactoring we'll need to do at some point.